packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -396,6 +403,8 @@ export class TeleReporterReceiver {
   private _absolutePath(relativePath?: string): string | undefined {
     if (relativePath === undefined)
       return;
+    if (relativePath.startsWith('/') || relativePath.startsWith('\\'))
```

Comment:
Let's pass `isUnderTest` option to `TeleReporterReceiver` constructor, and only assert under test?